kpops/components/streams_bootstrap/streams/streams_app.py

```diff
@@ -124,6 +125,19 @@ def add_extra_output_topic(self, topic: KafkaTopic, role: str) -> None:
     def helm_chart(self) -> str:
         return f"{self.repo_config.repository_name}/{AppType.STREAMS_APP.value}"
 
+    @override
+    async def destroy(self, dry_run: bool) -> None:
+        cluster_values = await self.helm.get_values(
+            self.namespace, self.helm_release_name
+        )
+        if cluster_values:
+            log.debug("Fetched Helm chart values from cluster")
+            name_override = self._cleaner.helm_name_override
+            self._cleaner.app = self.app.model_validate(cluster_values)
+            self._cleaner.app.name_override = name_override
+
+        await super().destroy(dry_run)
```

Comment:
don't we first have to destroy before running the cleaner?

Reply:
This is destroying the component 🤔 It calls `super().destroy()`, which destroys the component.

Reply:
yes, I am asking about the sequence. Here we're deploying the cleaner first, then destroying the parent app

Reply:
nvm, my bad